packages/backend/src/core/activitypub/ApDeliverManagerService.ts

```diff
@@ -157,7 +157,7 @@ class DeliverManager {
 	public async execute() {
 		if (!this.userEntityService.isLocalUser(this.actor)) return;
 
-		const inboxes = new Set<string>();
+		const inboxes = new Set<{inbox: string; isSharedInbox: boolean;}>();
```

Comment:
こういう `Set` の使い方はあまりしないというかしたくない

```suggestion
		// The value flags whether it is shared or not.
		const inboxes = new Map<string, boolean>();
```

とかはできない？